client/post-editor/edit-post-status/index.jsx

```diff
@@ -54,6 +61,7 @@ var EditPostStatus = React.createClass( {
 		stats.recordEvent( 'Changed Sticky Setting', stickyEventLabel );
 
 		actions.edit( { sticky: ! this.props.post.sticky } );
+		this.props.toggleStickyStatus( this.props.post.sticky );
```

Comment:
Summary:
```
do we want individual actions or a general edit action? etc

Artur Piszek [1:22 PM] 
Individual

Matías Ventura [1:22 PM] 
I think so too

Artur Piszek [1:22 PM] 
I'm advocating for individual as an abstraction

Matías Ventura [1:22 PM] 
specially since we may have a component outside of the editor that does just that one action

[1:23] 
but it could also complicate things a lot since a lot of those actions are connected

Artur Piszek [1:23 PM] 
And we already have `toggleStickyStatus`

[1:23] 
In component

[1:24] 
So I think in the end it should migrate from compoennt to actions

Matías Ventura [1:24 PM] 
I agree with that

Artur Piszek [1:24 PM] 
In actions.js they can use general edit action if we want

[1:24] 
But we want to be able to import just a single action

[1:24] 
Ok, so I will continue in that fashion
```